client/index.html

```diff
@@ -27,7 +27,6 @@
   <link rel="stylesheet" href="/bower_components/bootstrap-datepicker/dist/css/bootstrap-datepicker3.css" />
   <link rel="stylesheet" href="/bower_components/bootstrap-select/dist/css/bootstrap-select.css" />
   <link rel="stylesheet" href="/bower_components/bootstrap-switch/dist/css/bootstrap3/bootstrap-switch.css" />
-  <link rel="stylesheet" href="/bower_components/bootstrap-touchspin/src/jquery.bootstrap-touchspin.css" />
```

Comment:
It reflects the current dependencies, not caused by changes in this PR, but ..well, current :).

Related to https://github.com/ManageIQ/manageiq-ui-self_service/issues/81